python/samples/concepts/grounding/grounded.py

```diff
@@ -89,9 +89,7 @@ async def run_entity_extraction(kernel: Kernel, plugin_name: str, summary_text:
         topic="people and places", example_entities="John, Jane, mother, brother, Paris, Rome", input=summary_text
     )
 
-    return await kernel.invoke(
-        plugin_name=plugin_name, function_name="ExtractEntities", arguments=arguments
-    )
+    return await kernel.invoke(plugin_name=plugin_name, function_name="ExtractEntities", arguments=arguments)
```

Comment:
Random thought, this isn't super pertinent to this PR but when we have more time in the future, should we do a CQ spike on enabling mypy for concept samples? Would just be nice to have things as clean as possible...